packages/ast/src/transform/wast-identifier-to-index/index.js

```diff
@@ -110,8 +126,48 @@ function transformFuncPath(
           instrNode.args[0] = numberLiteralFromRaw(globalOffset);
         }
       }
+
+      /**
+       * Labels lookup
+       */
+      if (instrNode.id === "br") {
+        const [firstArg] = instrNode.args;
+
+        if (isIdentifier(firstArg) === true) {
+          // if the labels is not found it is going to be replaced with -1
+          // which is invalid.
+          let relativeBlockCount = -1;
+
+          // $FlowIgnore: reference?
+          instrPath.findParent(({ node }) => {
+            if (isBlock(node)) {
+              relativeBlockCount++;
```

Comment:
While this is not the best solution it's the safest. The alternative would be to add `enter` and `exit` hooks and simulate the label stack.